V.2.0.0 proofreading changes (#128)
uptane/deployment-considerations: glossary.md

In order to be considered Uptane-compliant, an implementation SHALL follow all of these rules as specified in the document.

> Note that, following the recommendations of {{RFC2119}}, imperatives of the type defined here "must be used with care and sparingly.  In particular, they SHALL only be used where it is actually required for interoperation or to limit behavior which has potential for causing harm (e.g., limiting retransmisssions)  

Reviewer: There's an extra `"` in here.

Author: @pattivacek see edit below. In the Standard, this quote was removed from the glossary, so I am doing the same here. Hence, the problem cited above is gone
```suggestion
Note that, following the recommendations of {{RFC2119}}, imperatives of the type defined here will be used only when essential for security.
```
